### ISSUE TITLE
Can't create new project from template

### ISSUE THREAD

**Issue by EliotSlevin** [expo/xde#74]
Making a new project from the tab bar navigation template hangs, and never extracts. 

https://puu.sh/te8zF/e010730e43.png

Extracts.js exits with non-zero code: 1

**Comment by wisetwo** [comment]
same here with xde-2.13.0.dmg

**Comment by wisetwo** [comment]
@EliotSlevin this solved my problem: 
`rm -fr ~/.exponent/starter-app-cache`
help from official member James :
"The app template might be corrupted on your computer for some reason. That command will clear it, and if you try to Create New Project again, XDE will download a new app template for you."

**Comment by dikaiosune** [comment]
@EliotSlevin sorry about the delay here -- do the steps wisetwo describes work to fix this issue for you?

**Comment by yurivyatkin** [comment]
I had the same problem today, and the solution of @wisetwo helped to fix it.

**Comment by jesseruder** [comment]
Talked with @yurivyatkin on slack. The problem is that the `request` library sometimes hangs on a url that has a redirect. Think we need to solve this by using something like https://github.com/mzabriskie/axios so that we can use the Chromium web stack.

**Comment by dikaiosune** [comment]
Looks like there are several issues open on the request repo about how redirects are followed. https://github.com/request/request/issues/2344 seems like a potential candidate to look at. It has to do with 301 redirects failing when using a proxy.

**Comment by jesseruder** [comment]
We're making it easier to retry the download and clear the cache. We're also planning on using https://github.com/mzabriskie/axios to use the Chromium network stack.

**Comment by inancgumus** [comment]
This works on my Macbook and doesn't work on my iMac. Hangs on 'Downloading project files' or sometimes on 'Extracting project files'.

**Comment by jesseruder** [comment]
@inancgumus can you do Help -> Clear Cache and try again?

**Comment by flyweights** [comment]
me too,  Network problems?

**Comment by inancgumus** [comment]
Actually, after 1 hour it worked. Then, it never hangs again. Interesting. I'll try it again.

**Comment by dikaiosune** [comment]
If you need to manually download the starter apps, there are instructions [in this gist](https://gist.github.com/jesseruder/a627c0dc0427f074d1aecba997c349ef). You can always extract the tarball yourself and open the project directly in XDE.

**Comment by n3ps** [comment]
Windows, fresh install. Hangs on "Extracting project files..."

**Comment by bensm1th** [comment]
Same. windows,  its hangs for ever on "Extracting project files..."

**Comment by eidy** [comment]
Windows Install - hangs forever on "Extracting project files..."

**Comment by junctioniv** [comment]
Same thing here. Windows Install - Hangs on "Extracting project files..." while trying to create a new blank project or on a Tab navigation project. The Extract.js function appears to be erroring out.

**Comment by encantador-coder** [comment]
Same here. Windows 10 install - hangs forever on "Extracting project files...". I tried creating a project using the Blank template.

**Comment by abozanona** [comment]
Same problem here. Windows 10.

**Comment by mweitzel2005** [comment]
Same problem here on Windows 10.

**Comment by pmathur08** [comment]
Same problem as everyone else...

**Comment by c00ler82** [comment]
Run Expo XDE as administrator helped me with "Extracting project files" issue.

**Comment by charlesvinette** [comment]
Hello everyone. Thank you for reporting the issue, I will be taking a look at what is happening under Windows.

**Comment by charlesvinette** [comment]
Hello everyone. I investigated the problem today and found that with Windows 10 Creator's update, Windows is handling sim links a little differently. 

I am still looking for a better fix, but for now I found that activating the Developer's mode on Windows 10 will allow the sim links to be created, and your downloaded templates should extract just fine.

 To do so, just type in "settings for developer" in the Cortana search box and check "Developer's mode". 

Have a great week!
